### PR TITLE
Add Text-To-Speech Beta samples

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: texttospeech/cloud-client/resources/hello.ssml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
```
Comment (remi) [inline texttospeech/cloud-client/resources/hello.ssml]: Simplify this :)

For out samples, let's just use simple SSML:

```xml
<speak>Hello there.</speak>
```

Per request from product :)

Note: I made the same comment on Python's PR and I updated Node.js code

File: texttospeech/cloud-client/src/test/java/com/example/texttospeech/SynthesizeTextIT.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+/**
+ * Tests for SynthesizeText sample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class SynthesizeTextIT {
+
+  private static String OUTPUT = "output.mp3";
+  private static String TEXT = "Hello there.";
+  private static String SSML = "<?xml version=\"1.0\"?>\n"
```
Comment (remi) [inline texttospeech/cloud-client/src/test/java/com/example/texttospeech/SynthesizeTextIT.java]: It's worth testing to make sure that this works fine with a simple `<speak>Hi</speak>`

(per product)

File: texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+          .build();
+
+      // Build the voice request
+      VoiceSelectionParams voice = VoiceSelectionParams.newBuilder()
+          .setLanguageCode("en-US") // languageCode = "en_us"
+          .setSsmlGender(SsmlVoiceGender.FEMALE) // ssmlVoiceGender = SsmlVoiceGender.FEMALE
+          .build();
+
+      // Returns [google.rpc.Code.INVALID_ARGUMENT] when not specified.
+      AudioEncoding audioEncoding = AudioEncoding.MP3; // MP3 audio.
```
Comment (kurtisvg) [inline texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java]: Same as above - just use `AudioEncoding.MP3` below.

File: texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+      try (OutputStream out = new FileOutputStream("output.mp3")) {
+        out.write(audioContents.toByteArray());
+        System.out.println("Audio content written to file \"output.mp3\"");
+      }
+    }
+  }
+  // [END tts_synthesize_ssml_file]
+
+  public static void main(String... args) throws Exception {
+    ArgumentParser parser = ArgumentParsers.newFor("SynthesizeFile").build()
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java]: I copied from their example and oddly if you move build to the end, it doesn't compile.

`.defaultHelp(true)` and `.description(...)`

File: texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+          .build();
+
+      // Build the voice request
+      VoiceSelectionParams voice = VoiceSelectionParams.newBuilder()
+          .setLanguageCode("en-US") // languageCode = "en_us"
+          .setSsmlGender(SsmlVoiceGender.FEMALE) // ssmlVoiceGender = SsmlVoiceGender.FEMALE
+          .build();
+
+      // Returns [google.rpc.Code.INVALID_ARGUMENT] when not specified.
+      AudioEncoding audioEncoding = AudioEncoding.MP3; // MP3 audio.
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java]: Fixed

File: texttospeech/cloud-client/pom.xml [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+    <!-- Parent defines config for testing & linting. -->
+    <parent>
+        <artifactId>doc-samples</artifactId>
+        <groupId>com.google.cloud</groupId>
+        <version>1.0.0</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <properties>
+        <maven.compiler.target>1.7</maven.compiler.target>
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply texttospeech/cloud-client/pom.xml]: Fixed

File: texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java (hunk 3 above)
Comment (nnegrey) [reply texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java]: Fixed

File: texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+      // Select the type of audio file you want returned
+      AudioConfig audioConfig = AudioConfig.newBuilder()
+          .setAudioEncoding(audioEncoding)
```
Comment (kurtisvg) [inline texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeFile.java]: `.setAudioEncoding(AudioEncoding.MP3)`

File: texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+          .setLanguageCode("en-US") // languageCode = "en_us"
+          .setSsmlGender(SsmlVoiceGender.FEMALE) // ssmlVoiceGender = SsmlVoiceGender.FEMALE
+          .build();
+
+      // Returns [google.rpc.Code.INVALID_ARGUMENT] when not specified.
+      AudioEncoding audioEncoding = AudioEncoding.MP3; // MP3 audio.
+
+      // Select the type of audio file you want returned
+      AudioConfig audioConfig = AudioConfig.newBuilder()
+          .setAudioEncoding(audioEncoding)
```
Comment (kurtisvg) [inline texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java]: `.setAudioEncoding(AudioEncoding.MP3)`

**Comment by kurtisvg** [comment]
@dpebot merge when green

**Pull request merged**